### PR TITLE
Updated the Visual Studio project

### PR DESCRIPTION
**Pull request by gunchleoc**
It took some time-consuming clicky fiddling to include the libraries, so I'd like to spare everybody else the pain...

No idea why this is also showing my last 3 commits that have already been merged *headdesk*

### CODE REVIEW

**Comment by akien-mga** [comment]
> No idea why this is also showing my last 3 commits that have already been merged

Seems to be that @Bertram25 did a rebase + merge instead of just merge the PR without rebase, so all commits got new hashes and are shown as committed by @Bertram25 (and authored by you): https://github.com/ValyriaTear/ValyriaTear/commits/master

So you'd have to rebase your branch with `git pull --rebase upstream master` (provided you have an upstream remote that points to https://github.com/ValyriaTear/ValyriaTear, if not `git remote add upstream https://github.com/ValyriaTear/ValyriaTear`) and force push the fixed branch.

**Comment by Bertram25** [comment]
@akien-mga thks!

**Comment by Bertram25** [comment]
@gunchleoc Also, it seems you added absolute paths, specific to your machine, which doesn't look right to me.
@authenticate Could you give him a hand about this?

**Comment by gunchleoc** [comment]
Oops! I never used Visual Studio before - I'm used to Linux and Bazaar. I'll definitely continue development on Linux, much easier.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ![Valyria Tear](https://raw.githubusercontent.com/ValyriaTear/ValyriaTear/master/data/boot_menu/valyria_logo.png)
 
-**Development blog:** http://valyriatear.blogspot.com
```
Comment (Bertram25) [inline README.md]: As I told @RyDroid not so long ago, adding two trailing space in a md file forces newlines. So here, you've broken the files by removing them.
Please do not touch trailing spaces in md files. Could you fix this?

**Comment by gunchleoc** [comment]
Whoever came up with that trailing whitespace syntax... bad idea.

All fixed, but I'm getting linker errors now. back to the drawing board.

**Comment by Bertram25** [comment]
Looks fine so far.

**Pull request merged**